doc/how_to_guides/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst

```diff
@@ -397,13 +397,199 @@ constraints of an individual ``moveit_msgs::msg::MotionPlanRequest`` are
+    pos_constraint_item1.header.frame_id = "world";
+    pos_constraint_item1.link_name = "panda_hand";
+    
+    // Set a constraint pose
+    auto target_pose_item1 = [] {
+       geometry_msgs::msg::PoseStamped msg;
+       msg.header.frame_id = "world";
+       msg.pose.position.x = 0.3;
+       msg.pose.position.y = -0.2;
+       msg.pose.position.z = 0.6;
+       msg.pose.orientation.x = 1.0;
+       msg.pose.orientation.y = 0;
+       msg.pose.orientation.z = 0;
+       msg.pose.orientation.w = 0;
+       return msg;
+    } ();
+    item1.req.goal_constraints.push_back(kinematic_constraints::constructGoalConstraints("panda_link8", target_pose_item1));
+
+The action client needs to be initialize:
+
+::
+
+    // MoveGroupSequence action client
+    using MoveGroupSequence = moveit_msgs::action::MoveGroupSequence;
+    auto client = rclcpp_action::create_client<MoveGroupSequence>(node, "/sequence_move_group");
+ 
+    // Verify that the action server is up and running
+    if (!client->wait_for_action_server(std::chrono::seconds(10))) {
+       RCLCPP_ERROR(LOGGER, "Error waiting for action server /sequence_move_group");
+       return -1;
+    }
+  
+Then, the request is created:
+
+::
+   
+    // Create a MotionSequenceRequest
+    moveit_msgs::msg::MotionSequenceRequest sequence_request;
+    sequence_request.items.push_back(item1);
+    sequence_request.items.push_back(item2);
+
+Create goal and planning options. A goal response callback and result callback can be included as well.
+
+::
+
+    // Create action goal
+    auto goal_msg = MoveGroupSequence::Goal();
+    goal_msg.request = sequence_request;
+ 
+    // Planning options
+    goal_msg.planning_options.planning_scene_diff.is_diff = true;
+    goal_msg.planning_options.planning_scene_diff.robot_state.is_diff = true;
+    // goal_msg.planning_options.plan_only = true;
+
+Finally, send the goal request and wait for the response:
+
+::
+
+    // Send the action goal
+    auto goal_handle_future = client->async_send_goal(goal_msg, send_goal_options);
+ 
+    // Get result
+    auto future_result = client->async_get_result(goal_handle_future.get());
+ 
+    // Wait for the result
+    if (future_result.valid()) {
+       auto result = future_result.get();  // Blocks the program execution until it gets a response
+       RCLCPP_INFO(LOGGER, "Action completed. Result: %d",  static_cast<int>(result.code));
+    } else {
+       RCLCPP_ERROR(LOGGER, "Action couldn't be completed.");
+    }
+
+If the motion need to be stopped mid trajectory, the action have to be cancel with:
+
+::
+
+    auto future_cancel_motion = client->async_cancel_goal(goal_handle_future_new.get());
+
+
+The ``pilz_robot_programming`` package for a `ROS 1 Python script
 <https://github.com/PilzDE/pilz_industrial_motion/blob/melodic-devel/pilz_robot_programming/examples/demo_program.py>`_
-that shows how to use the capability.
+shows how to use the capability.
```

Comment:
we can probably just remove this paragraph, unless you still find it helpful?

Reply:
The basics are all covered I think, I just left it because it has some methods that are not in ROS2 yet, like the **pause** command. Nevertheless these commands are part of the pilz_robot_programming, not the Pilz Industrial Motion Planner.

I will delete the paragraph to not create confusion.